_posts/projects/2015-10-4-epigraphing-19th-C.-novels.md

```diff
@@ -0,0 +1,48 @@
+(1) novel author,
+(2) novel title,
+(3) novel publication date,
+(4) novel publication location,
+(5) epigraph text,
+(6) epigraph attribution,
+(7) author birth and death years, and 
+(8) file provenance. 
+This information is useful both for exploring epigraph-usage and for examining broader questions about the circulation and printing of novels in the 19th century, and could be quite easily adapted to digital techniques like mapping. **So far we've collected 7,689 epigraphs (as of 11 Oct 2015).**
+
+
+
+#EXPERIMENTAL CHALLENGES / OPPORTUNITIES FOR COLLABORATION
+
+* ##Finding untagged epigraphs in TEI-XML Files
+The process of digitizing epigraphs transforms the layout of the page into code, and this conversion necessarily alters the legibility of the epigraph. What makes the epigraphs possible to extract from the digitized collections is that each epigraph has been tagged in the XML by a human transcriber. However, there are certain digitized novels from the collection (such as James Fenimore Cooper's _Wyandotte_) that have been found to be mistagged as "quotes" instead of "epigraphs" [1,2]. To account for this important source of error, the ratios of epigraph-to-quote tags have calculated so as to identify TEI files that may have untagged epigraphs. Based on spot checks, a back-of-the-envelope calculation suggests that 1% of the epigraphs in the corpus are currently untagged or incorrectly tagged.
+
+ * ##Name disambiguation
+One of the challenges in using the existing epigraph attributions to produce citation networks is that there is little standardization of how names were spelled in 19th-century texts. There are, for example, at least 8 different spellings of Shakespeare. To make matters worse, epigraph attributions formats are similarly unpredictable. Shakespeare's "Hamlet," for instance, might receive the following epigraph attributions: (1) Hamlet; (2) Shakespeare; (3) Shakespeare's Hamlet; (4) Hamlet, Shakespeare; (5) William Shakespeare; (6) William Shakespeare's Hamlet; and so forth. Merely separating the titles of the works (when present) from the author names (when present) is itself a very complex problem with no uniformly agreed upon solution. Such ambiguities must be resolved if one wishes to produce a definitive graphical model of epigraph attributions for our particular corpus. One potential solution is to crowd-source this work, as must be done with epigraph mistagging. We wish to open this corpus to the public for additional editing and future research. (Anyone interested in collaborating on this should send an email to [Aaron](http://aaronplasek.com).) However, another potential solution to this challenge is to perform this name disambiguation algorithmically. Such a project might make a very interesting undergraduate thesis project in computer science, math, or statistics. 
```

Comment:
I would shorten and tighten this up in general. You may want to include a Call for Participation, as it seems like you are looking for collaborators.